specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -0,0 +1,5445 @@
+        },
+        "delete": {
+          "$ref": "#/definitions/DateAfterCreation",
+          "description": "The function to delete the blob version"
+        }
+      },
+      "description": "Management policy action for blob version."
+    },
+    "DateAfterModification": {
+      "properties": {
+        "daysAfterModificationGreaterThan": {
+          "type": "number",
+          "multipleOf": 1.0,
+          "minimum": 0,
+          "description": "Value indicating the age in days after last modification"
+        },
+        "daysAfterLastAccessTimeGreaterThan": {
+          "type": "number",
+          "multipleOf": 1.0,
+          "minimum": 0,
+          "description": "Value indicating the age in days after last blob access. This property can only be used in conjunction with last access time tracking policy"
+        },
+        "daysAfterLastTierChangeGreaterThan": {
+          "type": "number",
+          "multipleOf": 1.0,
+          "minimum": 0,
+          "description": "Value indicating the age in days after last blob tier change time. This property is only applicable for tierToArchive actions and requires daysAfterModificationGreaterThan to be set for baseBlobs based actions. The blob will be archived if both the conditions are satisfied."
+        },
+        "daysAfterCreationGreaterThan": {
+          "type": "number",
+          "multipleOf": 1.0,
+          "minimum": 0,
+          "description": "Value indicating the age in days after blob creation."
+        }
+      },
+      "description": "Object to define the blob tiering action conditions. Properties daysAfterModificationGreaterThan, daysAfterLastAccessTimeGreaterThan and daysAfterCreationGreaterThan are mutually exclusive. The daysAfterLastTierChangeGreaterThan property is only applicable for tierToArchive actions which requires daysAfterModificationGreaterThan to be set, also it cannot be used in conjunction with daysAfterLastAccessTimeGreaterThan or daysAfterCreationGreaterThan."
+    },
+    "DateAfterCreation": {
+      "properties": {
+        "daysAfterCreationGreaterThan": {
+          "type": "number",
+          "multipleOf": 1.0,
+          "minimum": 0,
+          "description": "Value indicating the age in days after creation"
+        },
+        "daysAfterLastTierChangeGreaterThan": {
+          "type": "number",
+          "multipleOf": 1.0,
+          "minimum": 0,
+          "description": "Value indicating the age in days after last blob tier change time. This property is only applicable for tierToArchive actions and requires daysAfterCreationGreaterThan to be set for snapshots and blob version based actions. The blob will be archived if both the conditions are satisfied."
+        }
+      },
+      "required": [
+        "daysAfterCreationGreaterThan"
+      ],
+      "description": "Object to define the number of days after creation."
```

Comment:
the description for "DayAfterCreation" might not be so accurate, since "daysAfterLastTierChangeGreaterThan" is added to it as a child properties. This looks like an Object to define the  conditions for actions on blob snapshot and blob version.

Reply:
Updated description, IMO it is OK and  If you still have concerns kindly provide with the entire description